src/pypromice/process/L1toL2.py

```diff
@@ -66,24 +66,39 @@ def toL2(
     except Exception:
         logger.exception('Flagging and fixing failed:')
 
-    if ds.attrs['format'] == 'TX':
```

Comment:
Consider the how the "periods" parameter is interpret by the `persistence_qc`. I am not sure if sample rates are addressed correctly. This means that a 10-minutes raw dataset will be processed differently than hourly tx data.

Reply:
Although the doc of persistence_qc said `period is how many hours a value can stay the same without being set to NaN`, it was, as you suspected, number of persistent values that were counted.

I adapted the code to handle duration of persistent periods:
https://github.com/GEUS-Glaciology-and-Climate/pypromice/blob/eee1cbde9898adbc589c89c6775edb283cf3caeb/src/pypromice/qc/persistence.py#L168-L195

I updated the default threshold values to reflect this (but they are stil up for discussion):

```
# period is given in hours, 2 persistent 10 min values will be flagged if period < 0.333
DEFAULT_VARIABLE_THRESHOLDS = {
    "t": {"max_diff": 0.0001, "period": 0.3},
    "p": {"max_diff": 0.0001, "period": 0.3},
    'gps_lat_lon':{"max_diff": 0.000001, "period": 6}, # gets special handling to remove simultaneously constant gps_lat and gps_lon
    'gps_alt':{"max_diff": 0.0001, "period": 6},
    't_rad':{"max_diff": 0.0001, "period": 0.3},
    "rh": {"max_diff": 0.0001, "period": 0.3}, # gets special handling to allow constant 100%
    "wspd": {"max_diff": 0.0001, "period": 6},
}
```